### PR TITLE
Vet clinic database: query and update animals table

### PR DESCRIPTION
**Pull request by RashidMenhas**
- Use database transactions.
- Modify and delete data in SQL.

c1
c2
c2 1
c3
c4

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -9,3 +9,30 @@
  select * from animals where name !='Gabumon';
  select * from animals where weight_kg >=10.4 and weight_kg <=17.3;
 
+begin;
+update animals set species= 'unspecified';
+rollback;
```
Comment (Ghiftee) [inline queries.sql]: - [ ] According to the requirements, kindly ensure that after each transaction, you verify that the change was made, then you roll back the change and then verify that the change was undone. e.g:

``` UPDATE animals
   SET species = 'unspecified';  - make change

   SELECT species from animals; - verify that change was made
   ROLLBACK; - rollback change

   SELECT species from animals; - verify that change was undone
```

File: queries.sql [hunk 2]
```diff
@@ -9,3 +9,30 @@
...
+
+ begin;
+ update animals set species= 'digimon' where name like '%mon';
+ update animals set species='pokemon' where species is null;
+ commit;
```
Comment (Ghiftee) [inline queries.sql:23]: - [ ] According to the requirements, kindly write a query which verifies that the change was made and persists after the commit. e.g

```sql
 SELECT species from animals; -- verify that change was made
   COMMIT;
   SELECT species from animals; -- verify that change persists after commit
```

File: queries.sql [hunk 3]
```diff
@@ -9,3 +9,30 @@
...
+
+ begin;
+ delete from animals;
+ rollback;
```
Comment (Ghiftee) [inline queries.sql:30]: - [ ] You are currently missing this requirement - ```After the rollback verify if all records in the animals table still exists.``` Kindly implement it

File: queries.sql [hunk 4]
```diff
@@ -9,3 +9,30 @@
...
+  rollback to first_savepoint;
+  update animals set weight_kg = weight_kg * (-1) where weight_kg <0;
+  commit;
+
+  select count(name) from animals;
+  select  count(name) from animals where escape_attempts = 0;
+  select AVG(weight_kg) from animals;
+  select MAX(escape_attempts) from animals where neutered = true or neutered = false;
+  select max(weight_kg), min(weight_kg) from animals group by species;
+  select AVG(escape_attempts) from animals where date_of_birth between '01-01-1990' and '01-01-2000' group by species;
```
Comment (Ghiftee) [inline queries.sql]: - [ ] Great work writing all the queries so far, for this last query, the requirement states those born between 1990 and 2000. This means the query should begin from ```'01-01-1990'``` and end at ```'31-12-2000'```

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello 👋🏽  @RashidMenhas

Good job working on the project so far! 
✔️ All files are putnin the right place
✔️ You added screenshots of your queries
However, there are some issues that you need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

- [ ] Great job using the microverse README template, however, you need to customize it so that it is more professional and gives more details about your project.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by TadesseDev — APPROVED** [review]
### Hi @RashidMenhas 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
ST3D


## Highlights
 - [ ] Good job making the changes requested by the previous code reviewer. ✅
 - [ ]  Correct GitFlow. ✅
 - [ ]  Good PR title/description. ✅
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**